src/components/pacbio/PacbioPoolSelectedList.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div data-type="selected-pool-list">
+    <div class="flex flex-col py-3">
+      <div class="pools">
+        <traction-list-group class="list-group">
```

Comment:
I would move the css classes at the bottom of the file into tailwind classes here. Then you can remove the list-group name.

Reply:
I've left this in for now as it seems to be connected to some other styles at the moment